source/nodejs/adaptivecards-aaf/src/invoke-activity.ts

```diff
@@ -28,7 +27,7 @@ export interface InvokeActivity {
 }
 
 export enum ActivityInvocationContext {
-    AutoRefresh,
+    Refresh,
```

Comment:
I think we should name this enum value "Automatic" (vs the other one that is "UserInteraction")

Reply:
Yeah sounds good, will change.

Reply:
Actually, if that's not much to ask - could you also rename the enum itself into **ActivityInvocationTrigger** and rename variables/function parameters of that type from **context** to **trigger**?

If you'd rather not I can always do it later.

Reply:
Updated with the changes